src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -155,6 +155,8 @@ public enum CassandraRelevantProperties
 
     RING_DELAY("cassandra.ring_delay_ms"),
 
+    BOOTSTRAPPING_FAT_CLIENT_TIMEOUT("cassandra.bootstrapping_fat_client_timeout_ms"),
+
```

Comment:
I think we should name this something like FAILED_BOOTSTRAP_TIMEOUT, I don't want to expose 'fat client' to users since that is not immediately understandable and we have moved to calling them 'gossip only clients'